h/static/scripts/controllers/character-limit-controller.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+function CharacterLimitController(element) {
+  var counter = element.parentElement.querySelector(
+    '.js-character-limit-counter');
```

Comment:
Sure, I moved the controller onto another DOM element as well, but it's probably a good CSS class naming pattern to use `"js-foo-controller"` and then `"js-foo-controller-*"` isn't it?